wagtail/wagtailadmin/templates/wagtailadmin/chooser/_browse_results.html

```diff
@@ -4,7 +4,7 @@
 {% include "wagtailadmin/shared/breadcrumb.html" with page=parent_page choosing=1 %}
 
 {% if pages %}
-    {% include "wagtailadmin/pages/listing/_list_choose.html" with allow_navigation=1 orderable=0 pages=pages parent_page=parent_page %}
+    {% include "wagtailadmin/pages/listing/_list_choose.html" with allow_navigation=1 orderable=0 choosing=1 pages=pages parent_page=parent_page %}
```

Comment:
I think I must have added that there by mistake, or forgot to remove it after taking out the code that was making use of it. I'll revert that change.